defaults/main.yml

```diff
@@ -1,42 +1,33 @@
 ---
```

Comment:
Can you add comments to the variables?

Reply:
updated the comments in the code and the README as well.
(as noted in #4, planning to extend these in further iterations, especially regarding SSL and module support)